es-ds-docs/nuxt.config.ts

```diff
@@ -0,0 +1,50 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+    compatibilityDate: '2024-04-03',
+
+    // https://nuxt.com/docs/getting-started/styling#the-css-property
+    css: [
+        '@energysage/es-bs-base/scss/bootstrap.scss',
```

Comment:
Part of what Nuxt Layers allows is to specify this sort of thing in the design system package, rather than in downstream repos. Is there a reason you put the options other than `extends` here rather than in `es-ds-components`?

Reply:
Also noting that the previous design system imported `bootstrap.import` rather than just `bootstrap`. I'm not sure if there's implications to that.

Reply:
Fair! Having the es-bs-base include come for free with the es-ds-components Nuxt layer would be easier.

Not sure of the difference between `bootstrap` and `bootstrap.import` - I see a `@forward` directive in `bootstrap.import`, but not sure that's necessary as I don't think we need to [make any variables or mixins available](https://sass-lang.com/documentation/at-rules/forward/#:~:text=The%20%40forward%20rule%20loads%20a,forward%20%22%22%20.) in any SCSS we write "after" the inclusion of bootstrap as there is none, it's passed straight to Nuxt to do a SASS build. In Vue components, I'd expect we still have to do the direct imports of `breakpoints` and `variables` as we did before.